packages/web-components/src/components/cbp-banner/cbp-banner.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component, Host, Prop, Element,  h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-banner',
+  styleUrl: 'cbp-banner.scss',
+})
+export class CbpBanner {
+
+  @Element() host: HTMLElement;
+
+  @Prop() color: string = 'infoBanner'; //intended to be used for different statuses (error, warning, etc)
+
+  handleDismiss() {
+    this.host.setAttribute('hidden', '');
+    //call event emitter for app to piggy back (need Event, EventEmitter ?)
+  }
+
+  render() {
+    return (
+      <Host aria-color={this.color}>
+        <div class='iconContainer'>
```

Comment:
Is this wrapper tag needed? All classes should be namespaced so they are less likely to have collisions. If you need a class it should be named `cbp-banner-icon-container` for example. Maybe BEM style like `cbp-banner__icon-container` - we should flesh this out and be consistent.